### ISSUE TITLE
Provide SHA2-512 / SHA3 hashes & GPG sigs

### ISSUE THREAD

**Issue by beerisgood** [LineageOS/lineage_wiki#19]
Please replace the MD5 hashes with secure ones: SHA2-512 and/ or SHA3
And please add GPG sigs so we can verify the integrity for the builds.

**Comment by anoxi** [comment]
As stated on the download page, md5sum is just provided to check the download was successful and you can already check the integrity of the builds here https://github.com/lineageos/lineage_wiki/blob/master/pages/meta/verifying_builds.md

**Comment by beerisgood** [comment]
Fair enough.

Now the next problem is, how to run the _keytool_ command on Windows without Java.
And in this case a GPG sig would be easy for all users to check.

**Comment by anoxi** [comment]
You're right, verifying build documentation is still a bit rough, need to work on that.
